lib/rack/default_content_type.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+module Rack
+
+  # DefaultContentType is a Rack middleware that supports
+  # providing a default Content-Type header for requests
+  # to specific paths that do not include a Content-Type
+  # header in the request.
+  #
+  # Default content types are configured by adding entries
+  # to the Rack::DefaultContentType.default_content_types
+  # class member hash.
+  #
+  # Each hash key must be a regular expression to test
+  # request paths for a match. The hash value is the content
+  # type string to add to the request.
+  #
+  # :reek:ClassVariable
+  # rubocop:disable Style/ClassVars
+  class DefaultContentType
+    def self.default_content_types
+      @@default_content_types ||= {}
```

Comment:
You should always avoid `@@`.  And indeed it should be superfluous here.  The receiving object, because of the `self.` qualifier, is the class itself.  So `@default_content_types` would refer to a variable on the class object, not one of its instances.

Reply:
Interesting... I hadn't realized there was such a thing as a "class instance variable" distinct from a "class variable." Cool!